### PR TITLE
DT-20-feature-dark-mode-home-page

### PR DESCRIPTION
**Pull request by Giang-st1-00**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+API_URL=https://services.dgu.io.vn
```
Comment (Duy-Tinh-Dev) [inline .env]: Delete

File: components/common/Comments/Comment.tsx [hunk 2]
```diff
@@ -80,18 +80,15 @@ const Comment: React.FC<CommentProps> = ({
     return (
         <div key={comment.id} className="flex my-2 select-none">
             <div className="w-full">
-                <div className="bg-white flex justify-start items-center p-6 rounded-2xl">
-                    <div className="w-[50px] p-3 rounded-2xl h-full bg-[#F5F6FA]">
-                        <p className="text-[#5357B6] text-[20px] text-center hover:cussor-pointer">
-                            {comment.countLiked}
-                        </p>
+                <div className="bg-white dark:bg-[#393939] text-[#5357B6] dark:text-[white] flex justify-start items-center p-6 rounded-2xl">
```
Comment (Duy-Tinh-Dev) [inline components/common/Comments/Comment.tsx]: text-white

File: components/common/Header/index.tsx [hunk 3]
```diff
@@ -9,41 +9,77 @@ import { useEffect, useState } from "react";
...
+    const [isAnimating, setIsAnimating] = useState(false);
+
+    const toggleTheme = () => {
+        if (theme == "light") {
+            setTheme("dark");
+        } else {
+            setTheme("light");
+        }
+
+        // Bắt đầu animation
```
Comment (Duy-Tinh-Dev) [inline components/common/Header/index.tsx]: Not comment vietnamese

**Comment by Duy-Tinh-Dev** [comment]
ok

**Comment by lamxuanthinh** [comment]
@Giang-st1-00  Please write description pull request

File: package.json
Comment (lamxuanthinh) [inline package.json:1]: I don't accept your change file package.json @Giang-st1-00

**Comment by lamxuanthinh** [comment]
oki

**Pull request merged**